flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolImplTest.java

```diff
@@ -576,7 +550,6 @@ public void testDiscardIdleSlotIfReleasingFailed() throws Exception {
 		try (TestingSlotPoolImpl slotPool = createSlotPoolImpl(clock)) {
 
 			setupSlotPool(slotPool, resourceManagerGateway, mainThreadExecutor);
```

Comment:
Why not replace it with `createAndSetUpSlotPool `?
Also for the test case `testCheckIdleSlot`.

Reply:
Because in this commit there is no `createAndSetUpSlotPool(Clock clock)` method in `SlotPoolImpl` related test cases. In future commits, I replace all the `createAndSetUpSlotPool` with `SlotPoolBuilder`.

Reply:
I see.